src/pages/privacy-policy.md

```diff
@@ -0,0 +1,165 @@
+---
+template: legal-page
+path: /privacy-policy
+title: Privacy Policy
+date: "Effective and last modified: January 1, 2020"
+intro: >-
```

Comment:
Doesn't matter too much, but is this intro not Markdown-able?

Reply:
It's a Markdown widget! :)